seal_fhe/src/encryptor_decryptor.rs

```diff
@@ -469,4 +566,58 @@ mod tests {
 
         assert_eq!(data, data_2);
     }
+
+    #[cfg(feature = "deterministic")]
```

Comment:
If you want to hold off on the changes to CI, we could have easier test coverage of this by just always defining these functions and always testing them, but only exporting them when given the feature flag. Something like
```rust
mod deterministic {
     // functions here
}
#[cfg(feature = "deterministic")]
pub use deterministic::*;